src/core/InterleavedBuffer.js

```diff
@@ -92,6 +94,14 @@ InterleavedBuffer.prototype = {
 
 		return new this.constructor().copy( this );
 
+	},
+
+	onUpload: function ( callback ) {
```

Comment:
I realize now that this is required by underlying THREE.js objects (specifically `/src/renderers/webgl/WebGLObjects.js`) ;)